src/api/PDFFont.ts

```diff
@@ -89,6 +101,13 @@ export default class PDFFont implements Embeddable {
     return this.embedder.widthOfTextAtSize(text, size);
   }
 
+  /**
+   * Returns basic metrics about the font
+   */
+  getFontMetrics() : FontMetrics {
+    return this.embedder.getFontMetrics();
+  }
```

Comment:
Since these font metrics are immutable, I'd prefer to set them as properties of the `PDFFont` (like what we do with `fontName`) rather than add a new method to access them. These fields can then be set in the constructor.

Reply:
I am happy to do that, but I would point out that if you set them as properties then you are executing this far more than if you only use it when you call the method -- there aren't many cases where you need those metrics, so by adding it as a method there is almost no cost to adding it, but if you add it to the constructor and set it as a property then we're setting it every single time the class is instantiated and using the extra memory.

Granted, it's a fairly small cost, but as you seem to be concerned with optimization I just thought I'd point out that putting it in a property is actually less optimal than how I did it =]

Reply:
Appreciate the observations about memory and performance. Performance is indeed something I am concerned about. But there are a number of priorities that have to be balanced. And in this case I think consistency trumps any potential performance issues.